README.md

```diff
@@ -16,7 +16,7 @@ and/or validation (`last-modified`, `etag`) information:
 
 For more information about Rack::Cache features and usage, see:
 
-https://rtomayko.github.com/rack-cache/
+https://rtomayko.github.io/rack-cache/
```

Comment:
This must be pretty old documentation. Maybe we should update it from this repo if possible?

Reply:
That makes sense. I can try to regenerate the docs

Reply:
Thanks that would be wonderful!